.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Kindly consider adding `test.md` in gitignore file on line 2 as it is required in the requirements that test.md file be ignored. Currently test.md file exist in your repo which should not be there.
 - A test.md file with the text "This file should be ignored by git".
 - Make sure that the last file is actually ignored by git and is not present in your GitHub repository.